manifests/params.pp

```diff
@@ -17,21 +17,28 @@
   $version = 'latest'
   $service = 'jenkins'
   $install_from_git = false
+  $install_from_pip = true
+  $install_from_pkg = false
   $git_revision     = 'master'
   $git_url          = 'https://git.openstack.org/openstack-infra/jenkins-job-builder'
 
   case $::osfamily {
     'RedHat', 'Amazon': {
-      $python_packages = [ 'python', 'python-devel', 'python-pip', 'python-argparse']
+      $python_packages = [ 'python', 'python-devel', 'python-pip', 'python-argparse', 'PyYAML']
```

Comment:
wouldn't not  installing it break the application? argparse is available through EPEL, which is where the jjb package was coming from.  that being said in theory yum should handle the dependencies so we shouldn't need this list at all